### DISCUSSION TITLE
Could you add the merge tickets API to the Zendesk client, please?

### DISCUSSION THREAD

**Discussion by aguptaupgrade** [cloudbees/zendesk-java-client#637] [Q&A] [answered]
Hi,
I was not able to find a way to call the following API using the Zendesk v2 client:
https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#merge-tickets-into-target-ticket

If this feature is missing would it be possible to add it, please?
Thank you 🙏🏽

**Comment by PierreBtz** [comment, marked as answer]
Indeed the API isn't implemented. PR welcome!

**Reply by PierreBtz** [reply]
https://github.com/cloudbees-oss/zendesk-java-client/issues/658
